plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -20,7 +20,7 @@ class ReleaseDoc < ActiveRecord::Base
     attr_reader :previous_resources
 
     def deploy
-      @previous_resources = resources.map(&:resource)
+      @previous_resources = resources.reverse.map(&:resource)
```

Comment:
We'd still want to deploy the configmaps before we deploy anything else etc ... so keep this ?